package.json

```diff
@@ -72,7 +74,9 @@
     "match-media-mock": "0.1.1",
     "prettier": "2.2.1",
     "react-test-renderer": "16.14.0",
-    "resize-observer-polyfill": "^1.5.1"
+    "resize-observer-polyfill": "^1.5.1",
+    "ts-jest": "^26.5.0",
+    "typescript": "^4.9.4"
```

Comment:
[clarification] These dependencies are installed by `@edx/frontend-build`. They shouldn't need to be installed explicitly in this package.json given `@edx/frontend-build` is installed.